app/assets/javascripts/react/components/presentational/Circulation/Kiosk.js

```diff
@@ -0,0 +1,38 @@
+import { connect } from 'react-redux';
+import React, { Component, PropTypes } from 'react';
+import Header from './Header';
+import Rooms from './Rooms';
+import ConnectedSlideGallery from '../../SlideGallery';
+
+class Kiosk extends Component {
+  render() {
+    return (
+      <div id="circulation_kiosk">
+        <div className="row">
+          <div className="col-md-12">
+            <Header {...this.props} />
+          </div>
+        </div>
+        <div className="row">
+          <div className="col-md-2">
+            <Rooms {...this.props} />
+          </div>
+          <div className="col-md-10">
+            <ConnectedSlideGallery {...this.props} />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+Kiosk.propTypes = {
+  slides: PropTypes.array.isRequired,
+  url: PropTypes.string.isRequired,
+  is_fetching_slides: PropTypes.bool.isRequired,
+  show_nav: PropTypes.bool.isRequired,
+  fetchSlides: PropTypes.func.isRequired,
+  scrollToSlide: PropTypes.func.isRequired
+};
```

Comment:
Since `Kiosk` is a "connected" component getting state from Redux, and it's manually rendering a `Rooms` component and passing in properties.. you need to have the `api` property included here so that `Rooms` is aware of it for the url. So `api: PropTypes.object.isRequired` is needed here.